common/app/views/fragments/amp/mainStylesheetAMP.scala.html

```diff
@@ -247,6 +253,25 @@
         transform: rotate(180deg);
     }
 
+    @* Block Quotes related styling below *@
+    .from-content-api blockquote {
+        margin: 1rem 0 1.75rem;
+    }
+
+    blockquote.quoted .inline-quote {
+        fill: #767676;
+        position: absolute;
+    }
+
+    blockquote.quoted .quoted__contents {
+        margin-left: 2rem;
+    }
+
+    blockquote.quoted .inline-quote svg {
+        width: 1.5rem;
+        height: 1.1rem;
+    }
+
```

Comment:
keep a watch on ending up with a huge list of unrelated css in one file, if it's possible to split it into logical groups, I think it would be useful.